src/renderer/renderer.cpp

```diff
@@ -339,6 +339,14 @@ data::Image createWaterSurfaceAnimImage() {
     static_cast<size_t>(WATER_MASK_HEIGHT * WATER_NUM_MASKS)};
 }
 
+
+auto getSize(SDL_Window* pWindow) {
```

Comment:
just curious on the lambda being converted to a function. It seems to be called only from one place.

Reply:
Ah yeah, it's indeed used only once on this branch - this is a preparation for some changes on #429, where we will need it in more than one place. I'll add a note to the commit message to clarify that!